opentelemetry-exporter-cloud-trace/src/opentelemetry/exporter/cloud_trace/__init__.py

```diff
@@ -377,10 +378,15 @@ def _format_attribute_value(value: types.AttributeValue) -> AttributeValue:
     elif isinstance(value, float):
         value_type = "string_value"
         value = _get_truncatable_str_object("{:0.4f}".format(value), 256)
+    elif isinstance(value, collections.Sequence):
+        value_type = "string_value"
+        value = _get_truncatable_str_object(
+            ",".join([str(x) for x in value]), 256
```

Comment:
nit, can use a generator
```suggestion
            ",".join(str(x) for x in value), 256
```

Reply:
Ohhh very nice!